.github/workflows/benchmark.yml

```
@@ -0,0 +1,39 @@
+    runs-on: ubuntu-latest
+    # if: >-
+    #   github.event.issue.pull_request != '' &&
+    #   (
+    #     contains(github.event.comment.body, '/benchmark')
+    #   )
+    permissions:
+      pull-requests: write
+    steps:
+      - run: lscpu
+      - name: Checkout
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)